js/views/chatHeadsVw.js

```diff
@@ -88,7 +103,12 @@ module.exports = baseVw.extend({
       );
     });
 
-    this.nextChatToShow = this.nextChatToShow >= totalChats ? this.nextChatToShow : this.nextChatToShow + this.showPerScroll;
+
+    this.nextChatToShow = this.nextChatToShow >= this.collection.length ? this.nextChatToShow : this.nextChatToShow + this.showPerScroll;
+  },
+
+  remove: function(){
```

Comment:
I don't think there's a place in the code currently where this view is ever removed. The remove function would still be called by the base view if the parent is removed, correct?